### ISSUE TITLE
Refreshing whole library doesn't refresh covers

### ISSUE THREAD

**Issue by ziomek64** [tachiyomiorg/tachiyomi#9481]
### Steps to reproduce

1. If cover  changed and you refresh whole library the cover won't update.

 I have the option checked in the settings. At least it says like that in my language. Library->Updates "Automatycznie odświeżaj metadane. Sprawdz czy są nowe okladki i opisy podczas aktualizowania biblioteki" , Loosely translated by me its "Update metadata. Check if there are new covers and descriptions when updating the library."

And I know covers changed because when I go to that manga and update just the manga manually the cover update. Most of my mangas come from MangaSee, but I doubt it's the problem. If this is some translation issue then I'm making a proposal to add updating of covers when refreshing the library.

### Expected behavior

1. Covers should update when refreshing whole library if the option in the settings is checked.

### Actual behavior

1. Covers won't update when refreshing whole library if the option in the settings is checked.

### Tachiyomi version

0.14.6

### Android version

10

### Device

SM-P550

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/tachiyomiorg/tachiyomi-extensions/issues/new/choose).
- [X] I have gone through the [FAQ](https://tachiyomi.org/help/faq/) and [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by curche** [comment]
If you changed your cover manually, then Refresh Covers on Library update will not change those covers

**Comment by ziomek64** [comment]
> If you changed your cover manually, then Refresh Covers on Library update will not change those covers

Of course, but I refreshed the whole library few times BEFORE and THEN went and tried manual updates.

**Comment by ziomek64** [comment]
And by manual updates I mean manual refresh, pull down to refresh

**Comment by manavchaudhary1** [comment]
You can manually update covers by following below steps.
SETTEINGS>ADVANCED>LIBRARY _(Refresh library covers)_.
It will refresh covers of all the entries in your app.
Hope it will resolve your problem.

**Comment by garvongithub** [comment]
Same here, it does not do it automactically even if the option to update automatically is ticked. You have to do it on manga details page manually or from advanced settings and then clear cover cache for them to show.
